scripts/create_source_tarball.sh

```diff
@@ -33,6 +33,6 @@ REPO_ROOT="$(dirname "$0")"/..
     mkdir -p "$SOLDIR/deps/downloads/" 2>/dev/null || true
     wget -O "$SOLDIR/deps/downloads/jsoncpp-1.8.4.tar.gz" https://github.com/open-source-parsers/jsoncpp/archive/1.8.4.tar.gz
     mkdir -p "$REPO_ROOT/upload"
-    tar czf "$REPO_ROOT/upload/solidity_$versionstring.tar.gz" -C "$TEMPDIR" "solidity_$versionstring"
+    tar --owner user:1000 --group user:1000 -czf "$REPO_ROOT/upload/solidity_$versionstring.tar.gz" -C "$TEMPDIR" "solidity_$versionstring"
```

Comment:
Shouldn't we use ``0:0``, so root? If extracted as a user, it'll be changed to the user/group id of the extracting user anyways and, if extracted as root, I'd expect root to be the owner and not the "random" user ``1000:1000``.

Reply:
@axic Damn, too late :-).

Reply:
I'll create another PR to change it, I think it should be ``0:0``.